server/neptune/workflows/internal/activities/github.go

```diff
@@ -204,3 +205,19 @@ func (a *githubActivities) FetchRoot(ctx context.Context, request FetchRootReque
 		LocalRoot: localRoot,
 	}, nil
 }
+
+type CompareCommitsRequest struct {
+	OldCommit string
+	NewCommit string
+}
+
+type CompareCommitsResponse struct {
+	IsDiverged bool
+}
```

Comment:
Request/Response is most likely going to change when this is implemented, this is just added to build the skeleton for now.